_posts/2017-11-2-SUDS.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: post
+title: SUDS
+date: 2017-11-2
+categories:
+  - Major Projects
+description: Shower Use Detection System
+image: https://csh.rit.edu/~bigdata/SUDS.jpg
+image-sm: https://csh.rit.edu/~bigdata/SUDS.jpg
+author: Seth Gower
+author-image: https://avatars3.githubusercontent.com/u/19225089
+author-bio: First year Computer Engineering student
+author-email: seth@sethgower.com
+author-social:
+  github: https://github.com/SethGower
+  linkedin: https://www.linkedin.com/in/sethgower/
+---
+I made SUDS for the CSH October Hackathon (10/27/17). All in all, during that day I probably put an entire 24 hours into setting up the hardware for the _single_ bathroom. That number includes some time after the Hackathon was over, and then taking into account about 8 hour that I slept during the Hackathon. I will admit that SUDS was not my original idea, it was a project from years ago that just didn't work anymore. The major difference between SUDS v1 and SUDS v2 is the fact that v2 is powered by a MCU (right now it is a Raspberry Pi B2, but will be changing to possibly an Arduino), instead of just a regular circuit driven by 2 AA batteries in the faceplate with the LEDs. This is just the start of SUDS, I am planning on rolling it out to the other bathrooms after I rethink the hardware design. I am planning on using an Arduino with an Ethernet shield so that the Arduino can connect to a server that will run a web app that has more functionality. For the app, I am planning on not only showing the live status of the showers (and possibly toilet stalls), but also generate statistics on how often and at what time the showers/stalls are in use to generate a heat map.
```

Comment:
You never really detail _what exactly SUDS is_, including the acronym and what it does. You sorta touch on it at the end, but it would be cool if you could start with a brief description of what it does, as this is a public-facing post.